packages/shared/clickhouse/scripts/dev-tables.sh

```diff
@@ -225,6 +225,8 @@ CREATE TABLE IF NOT EXISTS events
       updated_at DateTime64(6) DEFAULT now(),
       event_ts DateTime64(6),
       is_deleted UInt8,
+      bookmarked Bool DEFAULT false,
+      public Bool DEFAULT false,
```

Comment:
I would move those up next to version etc. That way we keep track of similar kinds of properties in the same part of the DDL.